Tests/test_integration.py

```diff
@@ -371,10 +371,10 @@ def __print_investigation_error(client, playbook_id, investigation_id, color=LOG
         entries = resp_json['entries']
         print_color('Playbook ' + playbook_id + ' has failed:', color)
         for entry in entries:
-            if entry['type'] == ENTRY_TYPE_ERROR:
-                if entry['parentContent']:
-                    print_color('\t- Command: ' + entry['parentContent'].encode('utf-8'), color)
-                print_color('\t- Body: ' + entry['contents'].encode('utf-8'), color)
+            if entry['type'] == ENTRY_TYPE_ERROR and entry['parentContent']:
+                print_color('- Task ID: ' + entry['taskId'].encode('utf-8'), color)
+                print_color('  Command: ' + entry['parentContent'].encode('utf-8'), color)
+                print_color('  Body:\n' + entry['contents'].encode('utf-8') + '\n', color)
```

Comment:
why did we merge this into one if?

Reply:
If I'm not mistaken, entries without `parentContent` are like this one:
![image](https://user-images.githubusercontent.com/38749041/71411432-e9816800-2651-11ea-80c6-ee2e1f45b0fe.png)
which doesn't carry useful information - if you think it's relevant, I'll revert it